tests/integration/scheduler.bats

```diff
@@ -25,7 +25,19 @@ function teardown() {
 	[[ "${lines[2]}" == *"runtime/deadline/period parameters: 42000/1000000/1000000" ]]
 }
 
+# Checks that scheduling policy can't be used together with specific CPUs.
+# As documented in sched_setattr(2):
+#
+#   ERRORS:
+#   ...
+#        EPERM  The CPU affinity mask of the thread specified by pid does not
+#        include all CPUs  in  the  system (see sched_setaffinity(2)).
+#
+# FIXME: it seems like we test kernel rather than runc, not sure what is the
```

Comment:
I wanted to test this specific error you metion in the previous lines, but I forgot to add a check the output msg:
https://github.com/opencontainers/runc/pull/4025/files#diff-84fffb149f8b86c903e09613cbebad12de596f4139d54e288388b1458e08b8ecR649-R651

Because there are more than one EPERM errors could be returned by the kernel, so users maybe confused when using scheduler together with CPUs. So could you help to add a output error msg check in this test and remove this FIXME? Thanks.

Reply:
So, this test is to check that runc emits a specific error message in case of EPERM? Yeah that makes sense.

Fixed.